### ISSUE TITLE
Accessibility: A11y_Live Chat Widget_Let’s Chat Form_Screenreader: When we expand the Combo box narrator doesn't announce correct number of items present inside the combo box.

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#3058]
Copied from #1750392.

[1750392_A11y_Live Chat Widget_Lets Chat Form_Screenreader.zip](https://github.com/microsoft/BotFramework-WebChat/files/4397050/1750392_A11y_Live.Chat.Widget_Lets.Chat.Form_Screenreader.zip)

“Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.” 

 

Environment Details:

Application Name: Live Chat Wedget

#URL:https://xxx.blob.core.windows.net/portals/democ2occatest4.html

Browser Versions: Microsoft Edge Version 80.0.361.69 (Official build) (64-bit)

Windows Version: Windows10

Chrome Version: Version 80.0.3987.149 (Official Build) (64-bit)

 

Repro Steps:

1)Launch the application url : https://xxx.blob.core.windows.net/portals/democ2occatest4.html

2)welcome to live chat support Screen get open.

4)Navigate to "Chat Now button" and click on it.

5)"Chat Form" screen get open.

6)Navigate to Combo Box and start narrator.

Actual:
When we expand the Combo box narrator doesn't announce correct number of items present inside the combo box.

Same issue is observed with NVDA.

Same Issue is observer with Chat Form in Suggested Actions Feature.

Expected:
When we expand Combo box. Narrator should announce correct number of item present inside the combo box.

User Impact:
Narrator user will not get correct information.

Recommendations:
Please refer to below URL which is repository for accessibility bug fixes.

https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

 

MAS Reference:

MAS 1.3.1

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
